### DISCUSSION TITLE
ssr cannot completely inject the resources that a page needs

### DISCUSSION THREAD

**Discussion by 202780181** [2392645016/vue#14] [Q&A] [answered]
Version
2.6.14

Reproduction link
https://github.com/tcstory/vue-ssr-issue

Steps to reproduce
npm run build
npm run start
visit http://localhost:3000/page1 with javascript enabled and disabled respectively
What is expected?
with javascript disabled, the corresponding css should be inject to html

What is actually happening?
the corresponding css is missed

**Comment by 2392645016** [comment, marked as answer]
What is actually happening?
the corresponding css is missed
